src/shared/meson.build

```diff
@@ -454,6 +455,7 @@ libshared_deps = [threads,
                   libcap,
                   libcrypt,
                   libdl,
+                  libfdisk,
```

Comment:
This means we get a new core dependency, no? While before only repart/homed linked to this, is it worth it for a single helper?

Reply:
well, i am pretty sure we'll grow more. like for example one that reads a uuid from libfdisk and parses it right away as sd_id128.

so far, we scared away for adding common helpers, but that's not sustainable. We should add common helpers, it's a good thing, not a bad thing.

Right now it's just one, but I am sure there will be more pretty soon.